redwood-treehouse/src/commonMain/kotlin/app/cash/redwood/treehouse/AppLifecycle.kt

```diff
@@ -18,7 +18,13 @@ package app.cash.redwood.treehouse
 import app.cash.zipline.ZiplineService
 import kotlin.native.ObjCName
 
-@ObjCName("FrameClockService", exact = true)
-public interface FrameClockService : ZiplineService {
+@ObjCName("AppLifecycle", exact = true)
+public interface AppLifecycle : ZiplineService {
+  public fun start(host: Host)
+
   public fun sendFrame(timeNanos: Long)
+
+  public interface Host : ZiplineService {
+    public fun requestFrame()
+  }
```

Comment:
This API is basically `MonotonicFrameClock`. It feels like next we should allow you to supply one of our instances of those rather than driving at 60fps. The implementation on the host-side would then be as simple as:
```kotlin
override fun requestFrame() {
  clock.withFrameNanos { nanos ->
    app.sendFrame(nanos)
  }
}
```